primeqa/pipelines/README.md

```diff
@@ -35,31 +40,37 @@ retriever.load()
 
 - Step 2:  Initialize the reader model. 
 
+Alternatively, we can use a reader based on ChatGPT.
+
 ```python
-reader = ExtractiveReader(model_name_or_path = model)
+reader = PromptGPTReader(api_key='API KEY HERE', model_name="gpt-3.5-turbo")
 reader.load()
 ```
 
+We can use a reader based on ChatGPT.```python
```

Comment:
the previous example was for chatGPT, this should be "We can also use a reader based on Flan-t5" ?

Reply:
We show we can use chatGPT and FLAN-T5

Alternatively, we can use a reader based on Flan-T5.

```
reader = PromptFLANT5Reader(model_name="google/flan-t5-xxl")
reader.load()
```